contributing/core/testing.md

```diff
@@ -0,0 +1,35 @@
+# Testing
+
+See the [testing readme](../..//test/readme.md) for information on writing tests.
```

Comment:
Should we consolidate the test readme to this file?

Reply:
Yeah that'd be good 👍

Reply:
Merged these two together in https://github.com/vercel/next.js/pull/40515/commits/22f8f851f702ff730294bb04c2e7446160cdcd1a#diff-5d428d569033a870db648131d8e7ffdb602f395e1ee5cedb3821d8145e33c9b0